docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,127 @@
-  proxy_server: teleport.example.com # replace with your proxy address
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 4.
-    der_ca_file: /path/to/cert
-  discovery:
-    base_dn: "*"
+  auth_token: 2239...c5b21
+  proxy_server: teleport.example.com:443
+
 auth_service:
   enabled: no
+ssh_service:
+  enabled: no
 proxy_service:
   enabled: no
-ssh_service:
-  enabled: no    
-```
-</TabItem>
-<TabItem scope={["cloud"]} label="Cloud">
-For Teleport Cloud, Windows Desktop Service should establish a reverse tunnel to
-the hosted proxy. This requires setting `proxy_server` to your cloud tenant and
-providing a join token.
 
-First, generate a join token with the following command:
-
-```code
-$ tctl tokens add --type=windowsdesktop
-```
-
-Copy the join token to a file on the instance where you will run Windows Desktop
-Service, and then use the following configuration:
-
-```yaml
-version: v3
-teleport:
-  auth_token: /path/to/token
-  proxy_server: mytenant.teleport.sh # replace with your cloud tenant
 windows_desktop_service:
   enabled: yes
   ldap:
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 4.
-    der_ca_file: /path/to/cert
+    addr:     '10.10.1.50:636'
+    domain:   'windows.teleport.example.com'
+    username: 'WIN\svc-teleport'
+    server_name: 'windows-server-hostname'
+    insecure_skip_verify: false
+    ldap_ca_cert: |
+        -----BEGIN CERTIFICATE-----
+        MIIDnzCCAoegAwIBAgIQT/UIn+MT4aZC9ix/QuiV9zANBgkqhkiG9w0BAQsFADBi
+        ...
+        31qA4dO3if7RdikD9hVbiIF9jQ==
+        -----END CERTIFICATE-----
+
   discovery:
-    base_dn: "*"
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no    
+    base_dn: '*'
+  labels:
+    teleport.internal/resource-id: 42d8859c-60d0-4d7f-9767-bdd66b63fce6
 ```
-</TabItem>
-</Tabs>
-
-Start Teleport after updating `/etc/teleport.yaml`:
 
-```code
-sudo teleport start -c /etc/teleport.yaml
-```
+Click **Next**.
 
-## Step 7/7. Log in using Teleport
+Once you've saved `/etc/teleport.yaml`, start Teleport:
 
-### Create a Teleport user/role for Windows Desktop Access
+<Tabs>
+<TabItem label="System Service">
 
-In order to gain access to a remote desktop, a Teleport user needs to have the
-appropriate permissions for that desktop. For example, you can create a role
-that gives its users access to all Windows desktop labels and the
-`"Administrator"` user:
+This command assumes the `root` user. Prepend `sudo` otherwise.
 
-```yaml
-kind: role
-version: v5
-metadata:
-  name: windows-desktop-admins
-spec:
-  allow:
-    windows_desktop_labels:
-      "*": "*"
-    windows_desktop_logins: ["jsmith"]
+```code
+$ systemctl start teleport.service
```

Comment:
Do we assume the user has already created a systemd unit for teleport?

Reply:
No, we have tabs for both. As rendered:

![image](https://user-images.githubusercontent.com/2349184/201380724-8832bbfb-4d86-467d-b5f3-8701ab22968f.png)
![image](https://user-images.githubusercontent.com/2349184/201380764-02e7ed43-27ff-4afe-bf7e-872496d33de0.png)